src/MusicWheel.cpp

```diff
@@ -736,15 +737,51 @@ void MusicWheel::BuildWheelItemDatas( std::vector<MusicWheelItemData *> &arrayWh
 							RageColor colorSection = SECTION_COLORS.GetValue(iSectionColorIndex);
 							iSectionColorIndex = (iSectionColorIndex+1) % NUM_SECTION_COLORS;
 							// Add the section item
-							arrayWheelItemDatas.push_back( new MusicWheelItemData(WheelItemDataType_Section, nullptr, ssprintf("%d",sectionName), nullptr, colorSection, songs.size()) );
+							arrayWheelItemDatas.push_back( new MusicWheelItemData(WheelItemDataType_Section, nullptr, ssprintf("%d",sectionName), nullptr, nullptr, colorSection, songs.size()) );
 							// Add all the songs in this section
 							for (auto const& song : songs)
 							{
-								arrayWheelItemDatas.push_back( new MusicWheelItemData(WheelItemDataType_Song, song, ssprintf("%d",sectionName), nullptr, SONGMAN->GetSongColor(song), 0) );
+								arrayWheelItemDatas.push_back( new MusicWheelItemData(WheelItemDataType_Song, song, ssprintf("%d",sectionName), nullptr, nullptr, SONGMAN->GetSongColor(song), 0) );
 							}
 						} 
 					}
 					break;
+				case SORT_GROUP:
+					for( unsigned i=0; i< arraySongs.size(); i++ )
+					{
+						Song* pSong = arraySongs[i];
+						Pack* pPack = pSong->GetPack();

```

Comment:
Here and in many places of the PR.

I think since StepMania/ITGmania uses "Group" in the codebase for Pack, we should adhere to the same style. So instead of calling things `Pack`, let's call things `Group`.

I think in the same vein, I think we should consider calling it `Group.ini` instead of `Pack.ini` to be consistent (but this I'm more open to discussion).